packages/services/src/services/Translation/__tests__/Translation.test.js

```diff
@@ -104,18 +109,25 @@ describe('TranslationAPI', () => {
   });
 
   it('should return a json with a recent timestamp', async () => {
+    const mockDate = new Date('2019');
+    const _Date = Date;
+    global.Date = jest.fn(() => mockDate);
+    global.Date.now = _Date.now;
```

Comment:
Would you try: https://jestjs.io/docs/en/snapshot-testing#2-tests-should-be-deterministic

Reply:
I tried using that and many other variations before using this approach, but unfortunately this one was the only one that actually had any effect on `response`. Not sure what is causing it though, the implementations don't differ much from another.